src/main/resources/values/values/strings.xml

```diff
@@ -61,15 +61,15 @@
 
     <!-- Sources Menu -->
     <string name="sources_home">Home</string>
-    <string name="move_to_browse">To Browse</string>
-    <string name="move_to_latest">To Latest</string>
+    <string name="move_to_browse">To browse</string>
+    <string name="move_to_latest">To latest</string>
 
     <!-- Reader Menu -->
     <string name="no_pages_found">No pages found</string>
-    <string name="no_previous_chapter">There is no previous chapter</string>
+    <string name="no_previous_chapter">There is the first chapter</string>
```

Comment:
This and the next version of the string are taken from Tachiyomi, I would prefer to match Tachiyomi here

Reply:
```suggestion
    <string name="no_previous_chapter">There is no previous chapter</string>
```